src/mono/mono/eglib/glib.h

```diff
@@ -1506,4 +1507,11 @@ mono_qsort (void* base, size_t num, size_t size, int (*compare)(const void*, con
 #define g_try_realloc(obj, size) (g_cast (monoeg_try_realloc ((obj), (size))))
 #define g_memdup(mem, size) (g_cast (monoeg_g_memdup ((mem), (size))))
 
+/*
+ * Clock Nanosleep
+ */
+
+gint
+g_clock_nanosleep (clockid_t clockid, gint flags, const struct timespec *request, struct timespec *remain);
```

Comment:
Windows build are unhappy.  I suggest defining this under `ifdef HAVE_CLOCK_NANOSLEEP`.